src/defs.h

```diff
@@ -219,3 +256,119 @@ typedef struct {
     int index;
     int next[128];
 } trie_t;
+
+struct phi_operand {
+    var_t *var;
+    basic_block_t *from;
+    struct phi_operand *next;
+};
+
+typedef struct phi_operand phi_operand_t;
+
+struct Inst {
+    struct Inst *next;
+    int idx;
+    opcode_t opcode;
+    var_t *rd;
+    var_t *rs1;
+    var_t *rs2;
+    int sz;
+    phi_operand_t *phi_ops;
+    char str[64];
+};
+
+typedef struct Inst Inst_t;
```

Comment:
Enforce snakecase and use "insn" to represent "instruction." That is, we get `insn_t`.

Reply:
> Let's unify our terminology. From now on, use insn as the abbreviation for "Instruction," rather than "inst."

So the "inst_t" is going to be "insn_t", right?

Reply:
> So the "inst_t" is going to be "insn_t", right?

Exactly.